neurolang/probabilistic/dichotomy_theorem_based_solver.py

```diff
@@ -235,53 +236,4 @@ def _maybe_reintroduce_head_variables(ra_query, flat_query, unified_query):
 
 
 def solve_marg_query(rule, cpl):
-    """
```

Comment:
Moved to `query_resolution` module, to be shared by other dichotomy solver. Should maybe go back to the full dichotomy solver module if we remove the small dichotomy solver